hack/iam-manager.yaml

```diff
@@ -627,22 +639,6 @@ subjects:
   namespace: iam-manager-system
 ---
 apiVersion: v1
-data:
-  aws.MasterRole: masters.cluster.k8s.local
-  aws.accountId: "000011112222"
-  iam.managed.permission.boundary.policy: iam-manager-permission-boundary
-  iam.managed.policies: shared.cluster.k8s.local
-  iam.policy.action.prefix.whitelist: s3:,sts:,ec2:Describe,acm:Describe,acm:List,acm:Get,route53:Get,route53:List,route53:Create,route53:Delete,route53:Change,kms:Decrypt,kms:Encrypt,kms:ReEncrypt,kms:GenerateDataKey,kms:DescribeKey,dynamodb:,secretsmanager:GetSecretValue,es:,sqs:SendMessage,sqs:ReceiveMessage,sqs:DeleteMessage,SNS:Publish,sqs:GetQueueAttributes,sqs:GetQueueUrl
-  iam.policy.resource.blacklist: kops
-  iam.policy.s3.restricted.resource: '*'
-kind: ConfigMap
```

Comment:
This is to avoid overwriting the config map values when user is trying to update iam-manager to the next version. Config map has to be installed separately so that next time when we run the installer it will not overwrite the previous values